backend/src/fhir_repository.py

```diff
@@ -94,10 +94,9 @@ def get_immunization_by_identifier(
             item = response["Items"][0]
             resp = dict()
             vaccine_type = self._vaccine_type(item["PatientSK"])
-            if not is_imms_batch_app:
-                vax_type_perms = self._parse_vaccine_permissions(imms_vax_type_perms)
-                vax_type_perm = self._vaccine_permission(vaccine_type, "search")
-                self._check_permission(vax_type_perm, vax_type_perms)
+            vax_type_perms = self._parse_vaccine_permissions(imms_vax_type_perms)
```

Comment:
The permissions currently stored in Redis are of the form `COVID19_CREATE`, but these functions are expecting something like `covid19:create` (the case doesn't matter but the delimiter does). We also have special values of the form `COVID19_FULL` (meaning `COVID19_CREATE` + `COVID19_UPDATE` + `COVID19_DELETE` + ...) which aren't handled at the moment.